lib/connectors/memory.js

```diff
@@ -582,7 +584,9 @@ Memory.prototype.update =
       }
     }, function (err) {
       if (!err) {
-        self.saveToFile(null, cb);
+        self.saveToFile(null, function(err) {
+          cb && cb(err, count);
+        });
```

Comment:
Cross-posting from #503: IIRC `saveToFile` passes the first arg as to the final callback, so this block can be probably simplified to `self.saveToFile(count, cb)`.